lib/msal-node/src/config/Configuration.ts

```diff
@@ -217,6 +225,24 @@ export function buildAppConfiguration({
         disableInternalRetries: system?.disableInternalRetries || false,
     };
 
+    // if client certificate was provided, ensure that at least one of the SHA-1 or SHA-2 thumbprints were provided
```

Comment:
can you group the checks related to clientCertificate together to avoid repeated checks of !!auth.clientCertificate?